### PR TITLE
Deus front dev

### PR DESCRIPTION
**Pull request by dnsenga**
Did substantial work on the front end (CSS) to add some animations.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,12 +7,35 @@
...
+      <h2>Background</h2>
+      <p>I was born in Kigali, Rwanda, in a family of two children, and I am the eldest.</p>
+      <p>Click here to see random pictures of me</p>
+      <button onclick="addRandomGreeting()">images</button>
+      <div id="image-container">
+	<img src="images/IMG-1944.jpg" alt="Image" height="170" width="170"/>
+      </div>
+      <h2>Experience</h2>
+      <ul>
+	<li>Google, STEP Intern, Summer 2020</li>
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: When creating lists/blocks/tables make sure to indent child elements so it is east to tell how things are nested. 

E.g:

```
<ul>
  <li>foo</li>
  <li>bar</li>
</ul>

``` 
See: https://google.github.io/styleguide/htmlcssguide.html#General_Formatting

Comment (ElanHR) [reply portfolio/src/main/webapp/index.html]: Nvm I see these have already been fixed in the followup commit. :)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,41 +1,214 @@
...
+				<div class="socials">
+					<li><a href="https://www.linkedin.com/in/deus-alleluia-nsenga-74703b144" target="_blank"><i class="fa fa-linkedin" aria-hidden="true"></i></a></li>
+					<li><a href="https://www.facebook.com/jvedbrown" target="_blank"><i class="fa fa-facebook-f" aria-hidden="true"></i></a></li>
+					<li><a href="https://www.instagram.com/deus_a_n/?hl=en" target="_blank"><i class="fa fa-instagram" aria-hidden="true"></i></a></li>
+					<li><a href="https://twitter.com/DeusNsenga" target="_blank"><i class="fa fa-twitter-square" aria-hidden="true"></i></a></li>
+					<li><a href="https://github.com/dnsenga" target="_blank"><i class="fa fa-github" aria-hidden="true"></i></a></li> 
+				</div>
+
+			</div>
+
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: nit: Please delete these extra newlines.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,41 +1,214 @@
...
+					<div class="text">Ivory Coast</div>
+				</div>
+			</div>
+			<div class="gallery-container h-2">
+				<div class="gallery-item">
+					<div class="image"> <img src="images/cathedral.JPG"></div>
+					<div class="text">Inside the Basilica</div>
+				</div>
+			</div>
+
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: ditto - extra lines

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,41 +1,214 @@
...
+				<div class="about-article">
+					<h4>Article</h4>
+					<a href="https://eipm.weill.cornell.edu/2019/08/please-welcome-deus-nsenga/">Englander Institute for Precision Medicine</a>
+				</div>
+			</div>
+		</div>
+	</section> 
+
+	<!-- Section: Experience --> 
+
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html]: ditto - extra lines

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,41 +1,214 @@
...
+         		</li>
+         		<li>
+         			<div class="period">Summer 2019</div>
+         			<div class="company-role">
+         				<p class="company">Weill Cornell Medicine: Englander Institute for Precision Medicine</p>
+         				<p>Software Development Intern</p>
+         			</div>
+         		</li>
+            	<li>
+              		<div class="period">Summer 2019</div>
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html:85]: Please align these divs with other child elements in the same block. This helps the reader quickly grasp the structure from the indentation. 

e.g.
```
<li>
  <div class="period">Summer 2019</div>
  <div class="company-role">
    <p class="company">ComzAfrica</p> 
    <p>Research and Development Intern</p>
  </div>
</li>
```

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -1,41 +1,214 @@
...
+              		<div class="company-role">
+                     	<p class="company">ComzAfrica</p> 
+                  		<p>Research and Development Intern</p>
+                	</div>
+            	</li>
+            	<li>
+              		<div class="period">Summer 2018</div>
+              		<div class="company-role">
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html:93]: ditto - indentation

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -1,41 +1,214 @@
...
+                     	<p class="company">BKTechouse</p> 
+                  		<p>Software Development Intern</p>
+                	</div>
+            	</li>
+            	<li>
+              		<div class="period">Spring 2018</div>
+              		<div class="company-role">
```
Comment (ElanHR) [inline portfolio/src/main/webapp/index.html:100]: ditto - indentation

**Review by ElanHR — CHANGES_REQUESTED** [review]
Just a few comments on newlines and indentation

File: portfolio/src/main/webapp/index.html (hunk 6 above)
Comment (dnsenga) [reply portfolio/src/main/webapp/index.html:93]: Thank you

**Review by ElanHR — APPROVED** [review]

**Pull request merged**